libevmasm/Assembly.cpp

```diff
@@ -24,6 +24,7 @@
 
 #include <libevmasm/CommonSubexpressionEliminator.h>
 #include <libevmasm/ControlFlowGraph.h>
+#include <libevmasm/Disassemble.h>
```

Comment:
I guess this is leftover now, or does anything really depend on it :-)? But I imagine, if it's unnecessary, this will be among @cameel's hotfixes :-).

Reply:
No, but nice that you caught it :P